app/views/draws/show.html.erb

```diff
@@ -19,34 +19,32 @@
   <%= render 'ungrouped_report' %>
 </div>
 <% if policy(@draw).intent_summary? %>
```

Comment:
I don't even know if we need this policy or the `Intent Actions` headline, but it's your call.